.github/workflows/test-ibm-podman-dns.yaml

```
@@ -0,0 +1,130 @@
+        if: ${{ !cancelled() }}
+        run: |
+          set -x
+          sudo docker run --rm --privileged --network=host \
+            registry.fedoraproject.org/fedora:44 \
+            bash -c '
+              dnf install -y --quiet podman
+              echo "=== podman pull --userns=host ==="
+              podman pull quay.io/centos/centos:stream9-minimal && echo "USERNS=HOST PULL: OK" || echo "USERNS=HOST PULL: FAILED"
+              echo "=== podman build --userns=host --network=host ==="
+              cat > /tmp/Containerfile <<DOCKERFILE
+FROM quay.io/centos/centos:stream9-minimal
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)